test/e2e/specs/site-editor/site-editor-url-navigation.spec.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Site editor url navigation', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'twentytwentyone' ),
+			// Fallback clean-up in case of failure.
+			requestUtils.deleteAllPosts(),
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
```

Comment:
How about cleaning up in `afterEach` so that we don't have to do the same in each test below?

Reply:
Normally this wouldn't be needed if the testing environment is recreated, but I'm not sure about that. Locally it doesn't so it kept the created entities.

What do you think it's better?
1. `afterEach` even we don't create all the type of entities in each test
2. remove it completely from an `after` block

Reply:
We share the same WordPress instance for all tests, unfortunately. I would love to have isolated testing environments too, but that's difficult for WordPress environment I guess 🤔 .

Normally, cleaning up things in `afterEach` is better, because even if the test fails it could still run the necessary tasks to clean up the state. And since that we use `requestUtils`, which calls REST API internally, it should be fast even if we are over-cleaning.